templates/controllers/grid/settings/user/form/userDetailsForm.tpl

```diff
@@ -13,7 +13,7 @@
 		// Attach the form handler.
 		$('#userDetailsForm').pkpHandler('$.pkp.controllers.grid.settings.user.form.UserDetailsFormHandler',
 			{ldelim}
-				fetchUsernameSuggestionUrl: {url|json_encode router=$smarty.const.ROUTE_COMPONENT component="api.user.UserApiHandler" op="suggestUsername" firstName="FIRST_NAME_DUMMY" lastName="LAST_NAME_DUMMY" escape=false},
+				fetchUsernameSuggestionUrl: {url|json_encode router=$smarty.const.ROUTE_COMPONENT component="api.user.UserApiHandler" op="suggestUsername" givenName="GIVEN_NAME_DUMMY" familyName="FAMILY_NAME_DUMMY" escape=false},
```

Comment:
Let's call these `*_PLACEHOLDER` instead of `_DUMMY`. In the UI Library, I've used a `__givenName__` syntax for URL replacement, though I'm not wedded to that syntax by any means.